### PR TITLE
Add Actions: inheritance

### PR DESCRIPTION
**Pull request by timriley**
[✨ **Preview** ✨](https://deploy-preview-144--hanami-guides.netlify.app/v2.0/actions/inheritance/)

### CODE REVIEW

File: content/v2.0/actions/inheritance.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+```ruby
+# slices/admin/action.rb
+
+module Admin
+  class Action < Hanami::Action
+  end
+end
+```
+
+These base class can be a useful place to put any config or behavior that you want for every action in your app or slice.
```
Comment (andrewcroome) [inline content/v2.0/actions/inheritance.md:82]:
```suggestion
These base classes can be a useful place to put any config or behavior that you want for every action in your app or slice.
```

File: content/v2.0/actions/inheritance.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+## Leveraging inheratance
```
Comment (andrewcroome) [inline content/v2.0/actions/inheritance.md:84]:
```suggestion
## Leveraging inheritance
```

File: content/v2.0/actions/inheritance.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+      def authenticate_user!(request, response)
+        # halt 401 unless ...
+      end
+    end
+  end
+end
+```
+
+With this, any action requiring authentication could include the module.
```
Comment (andrewcroome) [inline content/v2.0/actions/inheritance.md:134]:
```suggestion
With this, any action requiring authentication can include the module.
```

**Review by andrewcroome — APPROVED** [review]
Reads great, thanks @timriley. Couple of tiny fix ups.

**Comment by timriley** [comment]
Thanks @andrewcroome!

**Pull request merged**